src/main/java/com/arangodb/springframework/repository/SimpleArangoRepository.java

```diff
@@ -360,7 +362,11 @@ private <S extends T> String buildFilterClause(final Example<S> example, final M
     private String buildPageableClause(final Pageable pageable, final String varName) {
         if (pageable == null) return "";
         Sort persistentSort = AqlUtils.toPersistentSort(pageable.getSort(), mappingContext, domainClass);
```

Comment:
with an unpaged pageable, the sort is still ignored, but that is another problem and a rare use case

Reply:
In `Pageable#unpaged` default sort method implementation have `Sort.unsorted()`. So I didn't  included sort for unpaged `Pageable` in `SimpleArangoRepository#buildPageableClause`

Reply:
`org.springframework.data.domain.Unpaged` is unsorted, so this fine IMHO